src/Entity/Document.php

```diff
@@ -0,0 +1,90 @@
+<?php declare(strict_types = 1);
+
+namespace WhiteDigital\DocumentGeneratorBundle\Entity;
+
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+use WhiteDigital\EntityResourceMapper\Entity\BaseEntity;
+use WhiteDigital\EntityResourceMapper\Entity\Traits\Id;
+
+#[ORM\Entity]
+#[ORM\Table(name: 'wd_document')]
+class Document extends BaseEntity
+{
+    use Id;
+
+    #[ORM\Column]
+    private ?string $type = null;
+
+    #[ORM\Column]
+    private ?string $contentType = null;
+
+    #[ORM\Column(type: Types::JSON)]
+    private ?array $sourceData = null;
+
+    #[ORM\Column(type: Types::JSON)]
+    private ?array $templateData = null;
```

Comment:
Aizdomājos, vai nevajag vēl vienu kolonnu - destinationFormat (XLSX/PDF/DOCX/....)

Reply:
nu es it kā to contentType jau pieliku, bet nu teorētiski var arī kā konstantu pielikt, kas tas ir

Reply:
contentType ir kvīts vai rēķins vai akits.
destinationFormat ir PDF vai XLSX vai DOCX - divas dažādas lietas

Reply:
nē - type ir kvīts, contentType ir application/pdf